python/perf-kernels/rmsnorm.py

```diff
@@ -153,7 +217,8 @@ def benchmark(M, N, provider):
         x = torch.randn(M, N, device='cuda', dtype=dtype)
         y = torch.zeros_like(x, device='cuda')
         n_rows, n_cols = x.shape
-        blk_size = triton.next_power_of_2(n_cols)
+        MAX_FUSED_SIZE = 65536 // x.element_size()
```

Comment:
Can you add some comments on this magic number?

Reply:
sorry, just saw your messages, anything great than MAX_FUSED_SIZE, we start to have spills